backend/backend.go

```diff
@@ -21,9 +21,15 @@ func Func(name string, fn func(context.Context, string) ([]byte, error)) Backend
 	return &backendFunc{fn: fn, name: name}
 }
 
+// Func creates a Backend from a function and allows users to add a defai;t map.
```

Comment:
Looks like a typo in the comment

Reply:
ty for the catch, submitting something to fix it =)